packages/synapse-react-client/src/lib/utils/theme/DefaultTheme.ts

```diff
@@ -8,6 +8,18 @@ const defaultMuiTheme: ThemeOptions = {
   typography: typographyOptions,
   palette: palette,
   components: {
+    MuiAlert: {
```

Comment:
Small tweaks to make MUI alert look like our custom FullWidthAlert built on top of Bootstrap. Icons + buttons probably won't be right, but this will be usable for inline alerts.